modepy/__init__.py

```diff
@@ -79,11 +79,11 @@
 __all__ = [
         "__version__",
 
-        "Shape", "Face", "Simplex", "Hypercube",
+        "Shape", "Face", "Simplex", "Hypercube", "TensorProductShape",
         "face_normal", "unit_vertices_for_shape", "faces_for_shape",
         "submesh_for_shape",
 
-        "FunctionSpace", "PN", "QN", "space_for_shape",
+        "FunctionSpace", "TensorProductSpace", "PN", "QN", "space_for_shape",
 
         "jacobi", "grad_jacobi",
         "simplex_onb", "grad_simplex_onb", "simplex_onb_with_mode_ids",
```

Comment:
This should also fix the README and the doc index. Both only mention simplices but not tensor products.

Reply:
Added some mention of tensor products in a782377. Also updated `examples/derivatives.py` to use the new and improved `modepy` functionality in the front page.